### PR TITLE
Feat/multi arch images

### PR DESCRIPTION
**Pull request by Lennart01**
## Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] The light- and dark-theme are both supported and tested
- [x] The design was implemented and is responsive for all devices and screen sizes
- [x] The application was tested in the most commonly used browsers (e.g. Chrome, Firefox, Safari)

### CODE REVIEW

File: src/routes/Homepage/Homepage.scss
Comment (brandstetterm) [inline src/routes/Homepage/Homepage.scss:1]: But why?

Comment (Lennart01) [reply src/routes/Homepage/Homepage.scss:1]: Autoformatter :D

Comment (brandstetterm) [reply src/routes/Homepage/Homepage.scss:1]: Please remove the changes from this PR :D

**Review by mateo-ivc — APPROVED** [review]
lgtm

**Review by mateo-ivc — APPROVED** [review]
:shipit:

**Comment by tibeer** [comment]
Does this mean we will not get multi-arch support?

**Comment by BenedictHomuth** [comment]
> Does this mean we will not get multi-arch support?

It will be supported with our next release. We just opened a new PR for it.  See here #4339
Sorry for closing this PR without a comment/link.

**Pull request closed without merging**